test/helpers/Fixtures.sol

```diff
@@ -108,7 +110,7 @@ contract DeploymentHelpers is Test {
         address permissionlessGate;
         address vettedGate;
         address parametersRegistry;
-        address parametersRegistryImpl;
+        address strikes;
```

Comment:
Why `parametersRegistryImpl` is removed?

Reply:
couldn't find a place where it's used